samtranslator/swagger/swagger.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, doc):
         self._doc = copy.deepcopy(doc)
         self.paths = self._doc["paths"]
         self.security_definitions = self._doc.get("securityDefinitions", {})
+        self.gateway_responses = self._doc.get('x-amazon-apigateway-gateway-responses', {})
```

Comment:
Define `_X_APIGW_GATEWAY_RESPONSES` above (line 18)

Reply:
Ugh, y'know, I had this originally, but I made it rhyme with the use of `"securityDefinitions"`.